dts/bindings/display/solomon,ssd16xx-common.yaml

```diff
@@ -49,6 +45,17 @@ properties:
       an external temperature sensor is connected to the controller.
       The value selects which sensor should be used.
 
+  rotation:
+    type: int
+    default: 0
```

Comment:
default needs justification

https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#rules-for-default-values

Reply:
Thanks for checking and pointing out @decsny. I looked at existing bindings to not re-invent the wheel. This example also lacks the justification: https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/bindings/display/ilitek%2Cili9xxx-common.yaml#L23

I really struggle to find a good justification as a description for this. It does not correspond to default values after reset, nor is it a manufacturer recommendation. 

The default makes sense for when not interested in the rotation property, and it helps to avoid breaking existing sdd16xx usages.

Can you help find a suitable description? Or do I need to remove the default if it is not possible to give a good justification?

Reply:
You can say rotation is off by default, for properties like this where one of the values corresponds to some kind of "none" meaning. The binding you linked is wrong for not clarifying this. Since you found it, please add a commit to fix it

Reply:
Thanks for the excellent suggestion. I've updated this and added a commit for ili9xxx.